packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -1157,16 +1149,15 @@ export default class AnalysisWebWorker {
 	 * @param {Paper}                      paper The paper to analyze.
 	 * @param {module:parsedPaper/structure.Node} tree  The tree to analyze.
 	 *
-	 * @param {Object}                             analysisCombination                 Which assessors and score aggregator to use.
+	 * @param {Object}                             analysisCombination                 Which assessors to use.
```

Comment:
It would be great if we take this opportunity to remove the `TreeAssessor` references here. So, instead of an `analysisCombination`, just have `assessor` as the argument of this function.

Also, the fields that are created in `setupTreeAnalysis` (e.g., `_contentTreeAssessor`, `_seoTreeAssessor`, `_relatedKeywordTreeAssessor`) can be removed, along with all commented code.